### PR TITLE
added Link to https://spdx.org/licenses

### PR DESCRIPTION
**Pull request by SpocWeb**
To better support non OSI / FSF Licenses as [discussed in Issue 9097](https://github.com/NuGet/NuGetGallery/issues/9097)

### CODE REVIEW

**Comment by joelverhagen** [comment]
Thanks @SpocWeb!

@agr, could you shepherd this PR with a review, merge, and deployment?

@clairernovotny, could you take a look at this wording? This is the page that will be enhanced: https://licenses.nuget.org/Foobar.

File: src/NuGet.Licenses/Views/License/UnknownLicense.cshtml [hunk 1]
```diff
@@ -4,4 +4,6 @@
     ViewBag.Title = "Unknown license";
 }
 
-<p>We have no information regarding the '@Model.LicenseName' license.</p>
\ No newline at end of file
+<p>We have no information regarding the '@Model.LicenseName' license.</p>
+<p>We only host licenses that are FSF and OSI-approved.</p>
+<p>For infos on other SPDX licenses please visit the <a href="https://spdx.org/licenses/">SPDX license list</a>.</p>
```
Comment (agr) [inline src/NuGet.Licenses/Views/License/UnknownLicense.cshtml:9]:
```suggestion
<p>For information on other SPDX licenses please visit the <a href="https://spdx.org/licenses/">SPDX license list</a>.</p>
```

**Comment by agr** [comment]
The change looks good, thanks. I will wait for @clairernovotny to approve the wording.

**Comment by clairernovotny** [comment]
Looks good after the suggestion about `infos` is committed.

**Review by agr — APPROVED** [review]

**Pull request merged**